src/importexport/musedata/internal/musedata.cpp

```diff
@@ -83,28 +83,35 @@ void MuseData::musicalAttribute(QString s, Part* part)
             }
         } else if (item.startsWith("X:")) {
         } else if (item[0] == 'C') {
+#if 0 // TODO
```

Comment:
I don't like this way of fixing warnings. We should either remove this code completely or fix it. My experience tells me that such TODOs can exist for years without any attempt to fix them

https://github.com/musescore/MuseScore/pull/8892

Reply:
Removing is fine by me. That code had not been touched in 10 years, seems a left over tehm the 1.x code moved to Github, to become MuseScore 2, the initial commit on Github. So really old and stale...

Reply:
I decided to choose not the easiest solution, but hopefully the most correct one, namely fixing the TODO right now. I've no idea what MuseData even is, but found the spec on the internet and tried to understand the code, and corrected it as far as I could. However, the MuseData module is probably the most broken of all; it does not have any unit tests and when I tried to import a MuseData file the result was corrupted in every measure and couldn't even be opened.

Reply:
Do you have any MuseData sample files?

Reply:
I grabbed this one: https://bitbucket.org/musedata/bach/src/master/bg/cant/0001/stage2/01/01 
That looks like a MuseData file.

Reply:
Thanks. It does open fine in MuseScore 1.3, saving as mscz and opening that in 3.6.2 reports only one single corruption, but opening the .md in 2.3.2 or 3.6.2 reports tons of corruptions (as does my backport of this chenges here to 3.x, that does though detect the correct timesig, 1.3 does too). It opens them though when ignoring these,